src/test/java/org/apache/commons/lang3/EnumUtilsTest.java

```diff
@@ -354,8 +360,21 @@ public void test_getEnumMap() {
     @Test
     public void test_getEnumMap_keyFunction() {
         final Map<Integer, Month> test = EnumUtils.getEnumMap(Month.class, Month::getId);
-        assertEquals("{1=JAN, 2=FEB, 3=MAR, 4=APR, 5=MAY, 6=JUN, 7=JUL, 8=AUG, 9=SEP, 10=OCT, 11=NOV, 12=DEC}", test.toString(),
-                "getEnumMap not created correctly");
+        final Map<Integer, Month> expected = new HashMap() {{
+            put(1, Month.JAN);
```

Comment:
Use normal code, please. This notation will generate compiler warnings since the diamond notation can't be used.
```
        final Map<Integer, Month> expected = new HashMap<>();
        expected.put(1, Month.JAN);
        ...
```

Reply:
Hi,
Thank you for the feedback. I've made the changes. Please let me know if there's anything else.